modules/hal_exception/src/hal_exception_config.hpp

```diff
@@ -37,10 +37,10 @@ namespace hal::exception {
 // a lot more memory since every assert or throw string will be placed in flash.. around ~70 bytes per statement
 #define ASSEMBLE_THROW_MSG(line, tag, message, exception_type, action) \
   "!E " "[" tag  ": " str(line) "]: " "THROW! "  message \
-  "; exception: " str(exception_type) " Action: " str(action) "\n\0"
```

Comment:
Very curious why you would remove the newline token here!

Reply:
Ah shit forgot that one